page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/properties/Configuration.java

```diff
@@ -1,9 +1,12 @@
 package ru.sbtqa.tag.pagefactory.properties;
 
 import org.aeonbits.owner.Config;
+import org.aeonbits.owner.ConfigFactory;
+import org.aeonbits.owner.Factory;
+import org.aeonbits.owner.Config.LoadType;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.aeonbits.owner.Config.LoadType'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=145&issues=AWg3rr9k1_eUtAp8780E&open=AWg3rr9k1_eUtAp8780E)

[ISSUE_KEY]: AWg3rr9k1_eUtAp8780E